llvm/lib/Object/ELF.cpp

```diff
@@ -799,6 +799,49 @@ ELFFile<ELFT>::getSectionAndRelocations(
   return SecToRelocMap;
 }
 
+template <class ELFT>
+static bool checkOffsets(const typename ELFT::Phdr &Phdr,
```

Comment:
I've not attempted to compare: are these two moved functions identical to the originals?